src/components/Drawer/__tests__/Drawer.spec.tsx

```diff
@@ -1,108 +1,104 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 import { Drawer } from '../Drawer';
 
 jest.mock('react-transition-group', () => {
     const FakeTransition = jest.fn(({ children }) => children({ state: 'entered' }));
     return { Transition: FakeTransition };
 });
+const onClickMock = jest.fn();
```

Comment:
we normally define this for each test or in the `beforeEach` block. To make it global can lead to errors, as the mock might not reset correctly between tests.